test/smoke/swappable_config/default_test.rb

```diff
@@ -1,4 +1,22 @@
-describe file('/etc/nagios/nagios.cfg') do
+vname =
+  if os.name == 'debian'
+    'nagios4'
+  elsif os.name == 'ubuntu'
+    case os.release.to_f
+    when 18.04
+      'nagios3'
+    when 20.04
```

Comment:
This should just be > 20.04 here so these tests continue to work on newer versions